.github/workflows/check.yml

```diff
@@ -68,7 +68,9 @@ jobs:
         uses: taiki-e/install-action@cargo-hack
       # intentionally no target specifier; see https://github.com/jonhoo/rust-ci-conf/pull/4
       - name: cargo hack
-        run: cargo hack --feature-powerset check
+        run: |
+            cargo hack --feature-powerset --exclude-features arc check
+            cargo hack --feature-powerset --features arc --at-least-one-of alloc,std check
```

Comment:
I don't think we need this if you apply the change that I suggest below.

Reply:
Yeah, that was intentional but probably a bad idea. Since arc requires unstable, probably it's better to make `["arc"]` the same as `["arc", "alloc"`]`